cmd/server-mux.go

```diff
@@ -441,6 +442,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 				RawQuery: r.URL.RawQuery,
 				Fragment: r.URL.Fragment,
 			}
+			lmsg := fmt.Sprintf("redirect for %s", u.String())
```

Comment:
We are not planning to add logging in this manner.. there are no Info logs in Minio and we would like to keep it that way for now. We are planning to bring in logging framework in a different manner at some time in future. 

If you need access logs you should use bucket notification we support all API calls to be logged on different targets - https://docs.minio.io/docs/minio-bucket-notification-guide